compiler/src/dotty/tools/io/VirtualFile.scala

```diff
@@ -38,12 +38,6 @@ class VirtualFile(val name: String, override val path: String) extends AbstractF
     this.content = content
   }
 
-  override def hashCode: Int = path.hashCode
```

Comment:
I already noticed that this will not work as interested. I will roll back this change and use unique paths for all the virtual source files created by the repl. Currently all paths are `<console>` or `rs$line$i` which the wrong memorized source file.